chains/ethereum/helpers.go

```diff
@@ -13,17 +13,3 @@ const (
 	NftTransferSignature  = "NFTTransfer(uint256,uint256,address,address,uint256,bytes)"
 	ErcTransferSignature  = "ERCTransfer(uint256,uint256,address,uint256,address)"
```

Comment:
Move constants to listner, perhaps directly in `getSubscriptions()`

Reply:
I've moved it into Listener, but I'm a fan of keeping them as constants rather than in the function.

Reply:
I think this should actually be in `events.go` :thinking: